testplan/testing/multitest/base.py

```diff
@@ -425,26 +423,39 @@ def get_test_context(self):
                 testcases_to_run = sorted_testcases
 
             if testcases_to_run:
-                if hasattr(self.cfg, "xfail_tests") and self.cfg.xfail_tests:
-                    for testcase in testcases_to_run:
-                        testcase_instance = ":".join(
-                            [
-                                self.name,
-                                suite.name,
-                                testcase.name,
-                            ]
-                        )
-                        data = self.cfg.xfail_tests.get(
-                            testcase_instance, None
-                        )
-                        if data is not None:
-                            testcase.__func__.__xfail__ = {
-                                "reason": data["reason"],
-                                "strict": data["strict"],
-                            }
+                for testcase in testcases_to_run:
```

Comment:
why are we making this change, i think the original code is fine.

Reply:
ah i just want to remove extra if statements